main.c

```diff
@@ -1,5 +1,5 @@
 #include "src/clings.h"
 
-int main() {
+int main(void) {
```

Comment:
why not int argc, char ** argv ?

Reply:
I have enabled the compiler flags `-Wall -Wextra -Werror -pedantic`

Now compilation is a little more rigid. Here, the `-Werror` flag embeds the `-Wunused-parameter` flag, which causes compilation to fail in the event of unused parameters.